package.json

```diff
@@ -46,77 +46,77 @@
   },
   "dependencies": {
     "csstype": "^2.6.0",
-    "deep-object-diff": "^1.0.4",
+    "deep-object-diff": "^1.1.0",
     "emotion": "^8.0.8",
-    "history": "^4.6.1",
-    "lodash": "^4.17.4",
-    "marksy": "^6.0.2",
-    "normalize.css": "^7.0.0",
-    "prismjs": "1.6.0",
+    "history": "^4.7.2",
+    "lodash": "^4.17.11",
+    "marksy": "^6.1.0",
+    "normalize.css": "^8.0.1",
+    "prismjs": "1.15.0",
     "react-emotion": "^8.0.8",
     "react-live": "^1.12.0",
-    "react-redux": "^5.0.5",
+    "react-redux": "^5.1.1",
     "react-transition-group": "1.2.1",
-    "react-typography": "^0.16.5",
-    "redux": "^3.0.4",
-    "redux-actions": "^2.0.3",
+    "react-typography": "^0.16.18",
+    "redux": "^4.0.1",
+    "redux-actions": "^2.6.4",
     "to-style": "^1.3.3",
-    "victory-core": "^15.2.0"
+    "victory-core": "^31.1.0"
   },
   "peerDependencies": {
-    "prop-types": "^15.6.0",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "prop-types": "^15.6.2",
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.0.0-beta.42",
-    "@babel/core": "^7.0.0-beta.42",
-    "@babel/plugin-proposal-class-properties": "^7.0.0-beta.42",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.42",
-    "@babel/polyfill": "^7.0.0-beta.42",
-    "@babel/preset-env": "^7.0.0-beta.42",
-    "@babel/preset-react": "^7.0.0-beta.42",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "^8.2.2",
-    "babel-jest": "^22.1.0",
-    "babel-loader": "^8.0.0-beta.2",
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-proposal-class-properties": "^7.2.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.2.0",
+    "@babel/polyfill": "^7.2.5",
+    "@babel/preset-env": "^7.2.3",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^7.0.0-0",
```

Comment:
This should be removed. `@babel/core` is the replacement after 7 when the `@babel` namespace replaced `babel-` in all the official Babel project packages.

Reply:
The latest version of babel-jest unfortunately still requires this bridge version as a peer dependency, so we'll need to leave in for now. Otherwise, all tests fail.

Reply:
huh, never mind, this must have been an extension of issues related to updating source-map. removed!